### DISCUSSION TITLE
Claude 3 OPUS as an LLM in autogen?

### DISCUSSION THREAD

**Discussion by shaleenrajkaur** [microsoft/autogen#2092] [General]
Is the feature currently added and if not is it going to be an expected feature in the upcoming version? thank you

**Comment by marklysze** [comment]
Hi @shaleenrajkaur, AutoGen requires an OpenAI-compatible API. Anthropic's API needs to be converted to that and this project can do that:
https://github.com/jtsang4/claude-to-chatgpt

Claude has been used with AutoGen using that library and LiteLLM.

This page may get you started, instead of Ollama you would use this library:
https://microsoft.github.io/autogen/docs/topics/non-openai-models/local-litellm-ollama

**Reply by ekzhu** [reply]
@shaleenrajkaur if you find this useful, you are welcome to contribute to our documentation about how to use Claude
